core/Plugin.php

```diff
@@ -513,6 +513,36 @@ public function isTrackerPlugin()
         return false;
     }
 
+    /**
+     * @param string $pluginName
+     * @return Date|null
+     * @throws \Exception
+     */
+    public function getPluginLastActivationTime($pluginName)
```

Comment:
@diosmosis could rename it to `getLastActivationTime()` and remove the `$pluginName` and instead use `$this->pluginName`?

Reply:
👍